factory/installScripts/chrome/install-chrome-version.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/node
```

Comment:
These `install-x-version.js` look pretty same-y, I considered making some shared utility, but i'm not sure the juice is worth the squeeze. There would be a bunch of strings that need to be passed in.